### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by xcixor**
#### Purpose of the PR
* Creates all the features required by a driver and a passenger to use the application
#### Tasks to be complete
* Show all rides
* Show the details of a ride
* Show the status of a ride
* Show ride history for both the driver and the passenger
* Enable ride requesting
* Enable ride confirmation (reject/accept)
#### Manual testing
* Navigate to the repo's main page via  [this link](https://github.com/xcixor/my-ride)
* Under the repo's name copy the clone's url
* Open a terminal and navigate to where you want the repo to be downloaded
* Type git clone 'the url you copied above' and press enter to download
* Checkout to the branch develop
* Navigate to the UI folder
* Open the index file with your favorite browser
* Start running!
* To preview on mobile in chrome inspect the page and the browser's inspection tool will provide 
   previews for various screens
* Or preview the site [here](https://xcixor.github.io/my-ride/)
#### Relevant pivotal tracker stories 
[user should be able to view available rides ](https://www.pivotaltracker.com/n/projects/2179559)
[User should be able to see the details of a ride and make a request. ](https://www.pivotaltracker.com/n/projects/2179559)
[A driver should be able to see the status of a ride](https://www.pivotaltracker.com/n/projects/2179559)
[A user should be able to see ride history ](https://www.pivotaltracker.com/n/projects/2179559)

### CODE REVIEW

File: UI/driver_dashboard.html [hunk 1]
```diff
@@ -27,8 +27,8 @@ <h2>Hi Driver welcome to your workspace</h2>
             <div id="dash-sidenav" class="sidenav">
                 <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
                 <a href="index.html">Home</a>
-                <a href="#">My Rides</a>
-                <a href="#">Profile</a>
+                <a href="driver_history.html">History</a>
+                <a href="passenger_dashboard.html">Passenger Profile</a>
                 <a href="index.html">Log out</a>
             </div>
             <span style="font-size:30px;cursor:pointer; padding: 20px;" onclick="openNav()">&#9776;</span>
```
Comment (JoshuaOndieki) [inline UI/driver_dashboard.html]: Inline CSS :x:

File: UI/driver_dashboard.html [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+        </div>
+        <div class="side-nav">
+            <div id="dash-sidenav" class="sidenav">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+                <a href="index.html">Home</a>
+                <a href="driver_history.html">History</a>
+                <a href="passenger_dashboard.html">Passenger Profile</a>
+                <a href="index.html">Log out</a>
+            </div>
+            <span style="font-size:30px;cursor:pointer; padding: 20px;" onclick="openNav()">&#9776;</span>
```
Comment (JoshuaOndieki) [inline UI/driver_dashboard.html]: Inline CSS :x:

**Review by JoshuaOndieki — CHANGES_REQUESTED** [review]

File: UI/driver_dashboard.html (hunk 2 above)
Comment (xcixor) [reply UI/driver_dashboard.html]: let me check it out

**Comment by artorious** [comment]
Hi, As you fix your `CSS` take a look at whats causing the overflow of some elements in small screen.
Earlier versions didn't have this bug. 
 
screenshot_2018-06-21-17-51-28
screenshot_2018-06-21-17-51-56
screenshot_2018-06-21-17-52-10
screenshot_2018-06-21-17-52-18

**Comment by xcixor** [comment]
#### Purpose of the PR
* Implement LFA feedback
#### Tasks to be complete
* Create user profile page
* Add more details about a ride - number plate
* Public driver profile with contacts
* passenger sees ride status accept or reject
#### Screenshots
##### User profile
feed1

##### Link to view more driver details
feed2

##### More ride's details
feed5


##### Driver's profile
feed3

##### Ride status
feed4

**Comment by JoshuaOndieki** [comment]
You could have opened a new PR for these changes specifically requested by the LFA.
Anyway, since you have already pushed to this PR and as I can see all requested features have been implemented, you're good to go :+1:

**Pull request merged**